lib/ansible/module_utils/crypto.py

```diff
@@ -36,11 +38,37 @@
     pass
 
 try:
+    import cryptography
     from cryptography import x509
     from cryptography.hazmat.backends import default_backend as cryptography_backend
     from cryptography.hazmat.primitives.serialization import load_pem_private_key
     from cryptography.hazmat.primitives import hashes
     import ipaddress
+
+    # Older versions of cryptography (< 2.1) do not have
```

Comment:
...do not have what?

Reply:
Good point, I guess I got sidetracked while typing that comment :D